proto/node/v1/node.proto

```diff
@@ -93,6 +93,9 @@ message GovernanceMessage {
     WormchainStoreCode wormchain_store_code = 14;
     WormchainInstantiateContract wormchain_instantiate_contract = 15;
     WormchainMigrateContract wormchain_migrate_contract = 16;
+
+    // Wormhole Relayer module
+    WormholeRelayerSetDefaultRelayProvider wormhole_relayer_set_default_relay_provider = 18;
```

Comment:
Going to have problems when you merge because 18-20 are taken by Circle Integration.

Reply:
Got it, will use 21